### ISSUE TITLE
[1.12.2] Crash related to Chococraft?

### ISSUE THREAD

**Issue by rookieEd1** [Xalcon/ChocoCraft3#32]
I was playing with a friend on a server and I was moving into new chunks on a Davinci's Vessel when i suddenly crashed each time when I tried to rejoin the server the client just crashed before being able to load into the game. 
From what I can tell from the crash log something weird is going on with the Chocobo in a chunk near where i was? It ended up having to ask my friend to tp me to him as i was loading in which took several tries and then resulting in doing a /kill to make sure i was dead at that spot and could respawn and load in.
Crash Log: https://pastebin.com/FVDpX60P
short version of crash (what i think is relevant not sure)
"Description: Ticking entity 
java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Float
    at net.xalcon.chococraft.common.entities.EntityChocobo.useStamina(EntityChocobo.java:299)
    at net.xalcon.chococraft.common.entities.EntityChocobo.regenerateStamina(EntityChocobo.java:553)
    at net.xalcon.chococraft.common.entities.EntityChocobo.func_70636_d(EntityChocobo.java:484)"

**Comment by rookieEd1** [comment]
Encountered again https://pastebin.com/mzY4PSm2 seems to be same kind of error when trying to use a sleeping bag from open blocks (while exploring new chunks)?
"Entity's Vehicle: ~~ERROR~~ NullPointerException: null"
